enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,302 @@
+an 120 Gb boot image. This is too long to meet our target of being able to install a whole cloud within the hour. One of
+the reasons this is slow is because we cannot do the image resizing after the copy, since it's writing directly to a
+sized disk. Once imported, creating the nodes is fast, since the other disks are flashed from the snapshot. For now, we
+asked the team to look into meeting the requirement that importing the image can take 5 minutes or less. Another bit
+we're working on is defining the implementation details for how the image will be imported from the COS bucket to the
+PowerVS service. Currently, the terraform API doesn't support importing an image from bucket. We are looking into if it
+is possible to update the terraform provider to do this with support for optional keys to support both private and
+public buckets. IBM Cloud Object Storage supports public buckets, but the terraform provider API will need an update to
+make keys optional in order to support this workflow.
+
+At this point, we are still determining whether this will happen as part of the installer or the terraform provider.
+
+2. Published CoreOS Images to the IBM Cloud Pre-Authorized Default Images This approach would involve outputting our
+CoreOS OVA images in a queue where they can be pre-loaded into the IBM Cloud so that they are available as soon as the
+Virtual Server instance is available. This is significantly quicker than the method above but has several drawbacks. For
+one, these default images need to be pre-authorized in order for them to be added to the IBM Cloud. This is currently a
+manual process owned by the Power VS team, so it cannot be completed by a user. Additionally, while we could build
+automation to solve this, it's unclear how would allow the user to point to the correct CoreOS image in the default
+list, since they would have to know the image name and details before their instance is created. Finally, we don't want
+to muddy up the default image list with an ever growing list of CoreOS options. This pruning would need to be automated
+some other mechanism would need to be in place to protect the user experience.
+
+#### DHCP
+The Power VS environment doesn't currently have a way of handling DHCP within the environment. We've asked their team to
+look into how they can provide this, since static IPs are currently not supported for IPI installs.
+
+The team has proposed a phased approach to DHCP support. The first stage would involve providing DHCP services by
+deploying a Centos (or possibly RHEL if a subscription model can be sorted out) VM to the Power VS service. This would
+not be hidden from the user's account, but they would not be charged for its use. The goal would be for this to be
+available for any application that needs DHCP, not just OpenShift.
+
+#### Load Balancing
+When it came to load balancer, the best option we came up with was to use the IBM Cloud [VPC
```

Comment:
@mkumatag @clnperez could we add the diagram that you have in your presentation explaining the load balancing topology with a small explanation here?

Reply:
Added this under the "Design Details" heading

Reply:
We could still probably use an explanation of the load balancing though.